<launch>
    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
        <rosparam file="$(find map_follower)/resources/cfg/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find map_follower)/resources/cfg/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find map_follower)/resources/cfg/local_costmap_params.yaml"  command="load" />
        <rosparam file="$(find map_follower)/resources/cfg/global_costmap_params.yaml" command="load" />

        <rosparam file="$(find map_follower)/resources/cfg/teb_local_planner_params.yaml" command="load" />

        <param name="base_global_planner" value="global_planner/GlobalPlanner" />
        <param name="planner_frequency" value="1.0" />
        <param name="planner_patience" value="5.0" />

        <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" />
        <param name="controller_frequency" value="5.0" />
        <param name="controller_patience" value="15.0" />

        <!-- Our carlike robot is not able to rotate in place -->
        <param name="clearing_rotation_allowed" value="false" />
    </node>

    <!-- <node pkg="map_follower" type="mapFollower.py" output="screen" name="map_follower" /> -->
</launch>
